tools/mtouch/Assembly.cs

```diff
@@ -94,6 +94,26 @@ public void CopyMdbToDirectory (string directory)
 				Application.UpdateFile (mdb_src, mdb_target);
 			}
 		}
+		
+		public void CopyMSymToDirectory (string directory)
+		{
+			string msym_src = FullPath + ".aotid.msym";
+			var dirInfo = new DirectoryInfo (msym_src);
+			if (!dirInfo.Exists) // got no aot data
+				return;
+			var subdirs = dirInfo.GetDirectories();
+			foreach (var subdir in subdirs) {
+				var destPath = Path.Combine (directory, subdir.Name.ToUpper ());
```

Comment:
Why `.ToUpper`?

If that's not required, you can use `Application.UpdateDirectory` to copy one directory to another recursively.